packages/fluent/scss/treeview/_variables.scss

```
@@ -0,0 +1,161 @@
+$kendo-treeview-item-padding-x-md: map-get( $kendo-spacing, 2 ) !default;
+$kendo-treeview-item-padding-x-lg: map-get( $kendo-spacing, 2 ) !default;
+/// Vertical padding of treeview items.
+/// @group treeview
+$kendo-treeview-item-padding-y-sm: map-get( $kendo-spacing, 2 ) - map-get( $kendo-spacing, thin ) !default;
+$kendo-treeview-item-padding-y-md: map-get( $kendo-spacing, 2 ) !default;
+$kendo-treeview-item-padding-y-lg: map-get( $kendo-spacing, 2 ) + map-get( $kendo-spacing, thin ) !default;
+/// Border width of treeview items.
+/// @group treeview
+$kendo-treeview-item-border-width: 0 !default;
+/// Border radius of treeview items.
+/// @group treeview
+$kendo-treeview-item-border-radius: var( --kendo-border-radius-md, 0 ) !default;
+
+/// Padding of the treeview filter element.
+/// @group treeview
+$kendo-treeview-filter-padding: map-get( $kendo-spacing, 2 ) !default;
+
+
+/// The sizes of the treeview.
+/// @group treeview
+$kendo-treeview-sizes: (
+    sm: (
+        font-size: $kendo-treeview-font-size-sm,
+        line-height: $kendo-treeview-line-height-sm,
+        item-padding-x: $kendo-treeview-item-padding-x-sm,
+        item-padding-y: $kendo-treeview-item-padding-y-sm
+    ),
+    md: (
+        font-size: $kendo-treeview-font-size-md,
+        line-height: $kendo-treeview-line-height-md,
+        item-padding-x: $kendo-treeview-item-padding-x-md,
+        item-padding-y: $kendo-treeview-item-padding-y-md
+    ),
+    lg: (
+        font-size: $kendo-treeview-font-size-lg,
+        line-height: $kendo-treeview-line-height-lg,
+        item-padding-x: $kendo-treeview-item-padding-x-lg,
+        item-padding-y: $kendo-treeview-item-padding-y-lg
+    )
+) !default;
+
+
+/// Background color of the treeview component.
+/// @group treeview
+$kendo-treeview-bg: transparent !default;
+/// Text color of the treeview component.
+/// @group treeview
+$kendo-treeview-text: var( --kendo-component-text, inherit ) !default;
+/// Border color of the treeview component.
+/// @group treeview
+$kendo-treeview-border: inherit !default;
+
+/// Background color of the treeview items.
+/// @group treeview
+$kendo-treeview-item-bg: transparent !default;
+/// Text color of the treeview items.
+/// @group treeview
+$kendo-treeview-item-text: var( --kendo-component-text, inherit ) !default;
+/// Border color of the treeview items.
+/// @group treeview
+$kendo-treeview-item-border: inherit !default;
+
+/// Background color of hovered treeview items.
+/// @group treeview
+$kendo-treeview-item-hover-bg: get-theme-color-var( neutral-20 ) !default;
+/// Text color of hovered treeview items.
+/// @group treeview
+$kendo-treeview-item-hover-text: get-theme-color-var( neutral-190 ) !default;
+/// Border color of hovered treeview items.
+/// @group treeview
+$kendo-treeview-item-hover-border: inherit !default;
+
+/// Background color of focused treeview items.
+/// @group treeview
+$kendo-treeview-item-focus-bg: $kendo-treeview-item-bg !default;
```

use css variable to enable customization ( ref to @TeyaVes comment in the Pager PR ). Do so for every reused variable.
```suggestion
$kendo-treeview-item-focus-bg: var( --kendo-treeview-item-bg, #{$kendo-treeview-item-bg} ) !default;
```

---

As far as I understood the idea when I spoke with @TeyaVes , we should apply this technique when we use variables from other components, so when the user changes the CSS variable in the other component it can be reflected in this component as well. Here's an example:

These variables:
```
$kendo-pager-item-padding-x: $kendo-button-padding-y-md !default;
$kendo-pager-item-hover-bg: $kendo-list-item-hover-bg !default;
```

Should be changed to:
```
$kendo-pager-item-padding-x: var( --kendo-button-padding-y-md, #{$kendo-button-padding-y-md}) !default;
$kendo-pager-item-hover-bg: var( --kendo-list-item-hover-bg, #{$kendo-list-item-hover-bg}) !default;
```

The reason for this is because we want when the user changes the CSS variable, for example `--kendo-list-item-hover-bg` the new color to be automatically applied to the `--kendo-pager-item-hover-bg`.

But in this case (in the case of the Treeview) we just want the initial value of the given SCSS variable to be equal to the value of another SCSS variable (from the same component) when the SCSS compiler compiles the CSS:
```
$kendo-treeview-item-focus-bg: $kendo-treeview-item-bg !default;
```

And from here on the user can customize the values further by using the following CSS variables:
```
--kendo-treeview-item-bg
--kendo-treeview-item-focus-bg
```

I don't know, I got even more confused while writing this comment :)
But if we decide that the correct approach is to always have to the SCSS variable to be assigned the CSS variable (which value represents another SCSS variable), it would mean that in the `_variables.scss` file we should almost never have to one SCSS variable to be assigned another SCSS variable which seems a bit strange to me. 
```
// NOT GOOD
$kendo-treeview-item-focus-bg: $kendo-treeview-item-bg !default;
```

---

Discussed this in an ad-hoc meeting with @joneff and @epetrow and we decided not to follow this approach as in this case, the value of the sass variable remains unknown ( value could come from a css or sass var ), thus the more clean approach is to stick to standard reuse of the sass variables ( without using the css var ).